### PR TITLE
Feature/UH2000 implement header and footer section

### PR DESCRIPTION
**Pull request by caleones**
image

### CODE REVIEW

**Comment by jesusdlacrz** [comment]
header and footer comply with the requirements, therefore, I authorize pull

File: header-footer.html
Comment (Edimar-star) [inline header-footer.html:1]: Overall, it is well laid out, I like the design and colors.

File: styles.css
Comment (Edimar-star) [inline styles.css:1]: I like how specific you are in the selectors, it makes it easier to understand what you are doing.

**Review by Edimar-star — APPROVED** [review]
I think it's a good job, I liked the use of selectors and colors in the project.

File: header-footer.html
Comment (jesusdlacrz) [reply header-footer.html:1]: very good, meets the requirements and makes good use of the space on the page.

**Review by jesusdlacrz — APPROVED** [review]
authorized pull, meets requirements and looks good aesthetically.

**Review by RDCelin — APPROVED** [review]
Aesthetically pleasing, dark blue works well as a primary color

**Review by Katkat04 — COMMENTED** [review]
I like your work, it's clean an responsive as requested. Only thing to noted, is your commits, you could have done more of it to see the progress.

**Pull request merged**